diffraction/calibration.py

```diff
@@ -0,0 +1,149 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
+# @author Mads Bertelsen (original version), Céline Durniak
+
+import numpy as np
+import scipp as sc
+import scippneutron as scn
+import re
+
+
+def load_calibration(filename, mantid_args={}):
+    """
+    Function that loads calibration files using the Mantid algorithm
+    LoadDiffCal. This algorithm produces up to three workspaces, a
+    TableWorkspace containing conversion factors between TOF and d, a
+    GroupingWorkspace with detector groups and a MaskWorkspace for masking.
+    The information from the TableWorkspace and GroupingWorkspace is converted
+    to a Scipp dataset and returned, while the MaskWorkspace is ignored for
+    now. Only the keyword paramters Filename and InstrumentName are mandatory.
+
+    Example of use:
+
+      from scipp.neutron.diffraction import load
```

Comment:
I think the `diffraction` submodule was removed from `scippneutron`...
see https://github.com/scipp/scippneutron/blob/main/docs/about/release-notes.rst#breaking-changes-1
does this still depend on it and do we need to add it back in?